templates/default/awstats_host.conf.erb

```diff
@@ -0,0 +1,16 @@
+Include "/etc/awstats/awstats.model.conf"
+LogFile="/usr/share/awstats/tools/logresolvemerge.pl /var/lib/awstats/logs/ftp-osl<%= "_ftp" if @ftp %>/<%= @log_file %> /var/lib/awstats/logs/ftp-chi<%= "_ftp" if @ftp %>/<%= @log_file %> /var/lib/awstats/logs/ftp-nyc<%= "_ftp" if @ftp %>/<%= @log_file %> |"
```

Comment:
So `LogFile` needs a lot of reworking. I realize that currently we only are doing awstats from the mirrors, but we want this to work else where eventually so you need to take that into account.

So here's what I'm thinking:
- Add a `mirror` true/false which will enable the `logresolvemerge.pl` logic
- Rename `@ftp` to `@vsftp`
- Add `%YYYY-2%MM-2%DD-2` to the end of the log files which is needed so that awstats know's how to deal with the date scheme